doc/user/interop.md

```diff
@@ -23,6 +23,8 @@ themselves are explained in the
 * [Threading and interop](#threading-and-interop)
 * [Embedded configuration](#embedded-configuration)
 
+Also see the separate document on [JRuby-compatible Java interop](jruby-java-interop.md).
```

Comment:
I would add a 3/4-rd/th level title to make it more distinct from all the links above.